Dispatcher/Controllers/ProfileViewController.swift

```diff
@@ -0,0 +1,94 @@
+import UIKit
+
+class ProfileViewController: UIViewController {
+
+    @IBOutlet weak var userProfileShadowView: UIView!
+    @IBOutlet weak var tableView: UITableView!
+    
+    var optionsArray: [ProfileOptionModel] = [
+        ProfileOptionModel(icon: UIImage(named: "gearWheel")!, text: "Setting", navigateTo: Constants.Segues.goToSettings),
+        ProfileOptionModel(icon: UIImage(named: "documents")!, text: "Terms & privacy", navigateTo: Constants.Segues.goToTermsAndPrivacy),
+        ProfileOptionModel(icon: UIImage(named: "logout")!, text: "Logout")
+    ]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        userProfileShadowView.layer.masksToBounds = false
+        userProfileShadowView.layer.shadowColor = UIColor.black.cgColor
+        userProfileShadowView.layer.shadowOpacity = 0.2
+        userProfileShadowView.layer.shadowOffset = CGSize(width: 0, height: 2)
+        userProfileShadowView.layer.shadowRadius = 3
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(UINib(nibName: Constants.NibNames.profileOption, bundle: nil), forCellReuseIdentifier: Constants.TableCellsIdentifier.profileOption)
+    }
+    
+    
+    @IBAction func editProfileButtonPressed(_ sender: Any) {
+        self.performSegue(withIdentifier: Constants.Segues.goToUpdateProfile, sender: self)
+    }
+    
+    override func prepare( for segue: UIStoryboardSegue, sender: Any? ) {
+        if segue.identifier == Constants.Segues.goToUpdateProfile {
+            let destinationVC = segue.destination as! ViewProfileViewController
```

Comment:
If you don't do any preparation here before navigation please remove.

Reply:
Already removed on the updated version (next PR)
@abenartz